### DISCUSSION TITLE
Welcome to SubwayTooter Discussions!

### DISCUSSION THREAD

**Discussion by tateisu** [tateisu/SubwayTooter#143] [General]
## 👋 Welcome! 
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rakoo** [comment]
# Thank you !

I've been trying Fediverse apps but all of them have similar ux patterns of big posts with huge buttons in your face, and little to no color configuration. Subway Tooter solves that by being information-dense and has a million settings to configure, I love it ! And multi-account is also really well made, contrary to other apps that usually box accounts into their own contexts. Thank you very  much for this app !!
